config/groups.yaml

```diff
@@ -727,3 +727,9 @@
     - "2833207"
     - "2637174"
   id: 190178
+  - name: "haskellsucks"
+  members:
```

Comment:
This isn't valid YAML. See the other entries in the file for how to format this correctly.

Reply:
I think I fixed it but I'm not entirely sure I've resubmit it correctly so do let me know if I've done something silly.
Much appreciated